.github/workflows/ci.yml

```diff
@@ -56,10 +56,10 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ env.CLCACHE_DIR }}
-          key: windows-vs2019-clcache-${{github.ref}}-${{github.sha}}
+          key: windows-2019-clcache-${{github.ref}}-${{github.sha}}
           restore-keys: |
-            windows-vs2019-clcache-${{github.ref}}-
-            windows-vs2019-clcache-refs/heads/main-
```

Comment:
I think we should just append a `-v2` at the end so next time we can just increment it (and we can keep count how many times the cache bugged out...)

Reply:
I was thinking that if it happens next time we can just return to the original, as by that time they will all clear :)